README.md

```diff
@@ -24,18 +24,34 @@ Install Node modules:
 npm install
 ```
 
-Install the docker container with the database:
+Start the docker containers:
 ```
-docker run --name rra-postgis \
--e "POSTGRES_PASSWORD=rra" \
--e "POSTGRES_USER=rra" \
--e "POSTGRES_DB=rra" \
--p 5432:5432 \
--d \
-mdillon/postgis
+docker-compose up
 ```
+Stop the docker containers with:
+```
+docker-compose stop
+```
+
+The containers will store the information within themselves. If the container is deleted all the information will be lost.
+When using the application locally [Minio](https://minio.io/) is used as cloud storage to simulate S3. It's interface will be available at `http://localhost:9000`.
+
+### Setup
+Both the database and the local storage need some setup.
+```
+npm run setup
```

Comment:
You have to setup the config before running that command. Will add instructions to the readme.